src/actions/global/globalactions/AddHaskellEvaluatorComponentGlobalAction.java

```diff
@@ -1,15 +1,32 @@
 package actions.global.globalactions;
 
 import actions.global.GlobalPaintAction;
+import paintcomponents.haskell.EvaluateHaskellPaintComponent;
 import ui.PaintPanel;
 
 public class AddHaskellEvaluatorComponentGlobalAction
 		extends GlobalPaintAction {
 
+	// currently cannot be evaluated by creating the line seg
+	// Error: typesystem.HaskellType cannot be cast to typesystem.JavaType
+
+	private Class classToCreate;
+
+	/**
+	 * @param classToCreate
+	 *            the classToCreate to set
+	 */
+	public void setClassToCreate(Class classToCreate) {
```

Comment:
Is this class Ever Used?

If not, remove classToCreate. This is not creating a class.